src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,16 @@
-//
+.PostInfo {
+  margin: 10px auto;
+  width: 90%;
+  border: 1px solid #000;
+  border-radius: 8px;
+  background-color: lightgray;
```

Comment:
Do not use color names, as such colors may be different in different browsers, use HEX codes

Reply:
That's, kind of, code from the original repo. Not my doing)
And, yeah I know that hex is better)